src/app/public/modules/auth-http-client/auth-interceptor.ts

```diff
@@ -14,12 +14,13 @@ import {
 } from '@angular/core';
 
 import {
+  from as observableFrom,
```

Comment:
Is the formatting off here?

Reply:
What do you mean?

Reply:
Nevermind. The `from` keyword was throwing me off. I now see that's the rxjs function and not the typescript import syntax.